.gitignore

```diff
@@ -1,2 +1,9 @@
 /sulu/
 /styleguide-globals.js
+
+# IDEs
```

Comment:
@nnatter okay for me but would recommend you add this kind of files to your [global gitignore](https://github.com/alexander-schranz/dotfiles/blob/master/composer/.gitignore) some open source project like [symfony](https://github.com/symfony/symfony/blob/v5.1.7/.gitignore) don't add IDE specific .gitignore and recommend you having this things in the global gitignore.